tests/unit/test_imbalance.py

```diff
@@ -0,0 +1,86 @@
+import os
+import shutil
+
+import pytest
+
+import tfs
+
+from omc3.scripts import luminosity_imbalance
+
+CURRENT_DIR = os.path.dirname(__file__)
+
+
+def test_result_tfs(_tfs_file):
+    res = luminosity_imbalance.main({'tfs': _tfs_file})
```

Comment:
also hint:
`res = luminosity_imbalance.main(tfs=_tfs_file)`

looks more function-like?

Reply:
I prefer the dict, it conveys more the idea that there is the entrypoint behind

Reply:
But the whole idea of the entrypoint is, that it is usable from python as a function :D 
That's why we have the entrypoint-wrapper